centreon/src/CentreonRemote/Domain/Service/ConfigurationWizard/ServerConnectionConfigurationService.php

```diff
@@ -277,4 +276,72 @@ protected function isRemote(): bool
     {
         return false;
     }
+
+    /**
+     * @param array<string, array<int, string[]>> $brokerInfos
+     *
+     * @throws \Throwable
+     *
+     * @return array<string, array<int, string[]>>
+     */
+    protected function saveCredentialInVault(array $brokerInfos): array
+    {
+        $kernel = Kernel::createForWeb();
+        /** @var ReadVaultConfigurationRepositoryInterface $readVaultConfigurationRepository */
+        $readVaultConfigurationRepository = $kernel->getContainer()->get(
+            ReadVaultConfigurationRepositoryInterface::class
+        );
+        /** @var FeatureFlags $featureFlagManager */
+        $featureFlagManager = $kernel->getContainer()->get(FeatureFlags::class);
+
+        /** @var WriteVaultRepositoryInterface $writeVaultRepository */
+        $writeVaultRepository = $kernel->getContainer()->get(WriteVaultRepositoryInterface::class);
+        $writeVaultRepository->setCustomPath(AbstractVaultRepository::BROKER_VAULT_PATH);
+
+        $vaultConfiguration = $readVaultConfigurationRepository->find();
+
+        if (
+            ! $featureFlagManager->isEnabled('vault')
+            || ! $featureFlagManager->isEnabled('vault_broker')
+            || $vaultConfiguration === null
+        ) {
+            return $brokerInfos;
+        }
+
+        foreach ($brokerInfos as $key => $inputOutput) {
+            $inputOutputName = null;
+            $credentialKey = null;
+            $credentialValue = null;
+            foreach ($inputOutput as $index => $row) {
+                if ($row['config_key'] === 'name') {
+                    $inputOutputName = $row['config_value'];
```

Comment:
No isset() on $row['config_value'] ? Are you sure this array contains this key at this location ?

Reply:
thanks ;)